### ISSUE TITLE
False positive unused import

### ISSUE THREAD

**Issue by emlyn** [borkdude/clj-kondo#876]
**version**

clj-kondo v2020.02.28

**problem**

False positive unused import with:
```clojure
(ns foo
  (:import java.net.URI))

(defn uri [s]
  (java.net.URI/create s))
```

**repro**
```
clj-kondo --lint - <<< '(ns foo (:import java.net.URI)) (defn uri [s] (java.net.URI/create s))'
<stdin>:1:18: warning: Unused import URI
linting took 31ms, errors: 0, warnings: 1
```

**Comment by emlyn** [comment]
Strangely, changing the class name slightly (from `java.net.URI` to `javanet.URI`) adds another false positive:
```
clj-kondo --lint - <<< '(ns foo (:import javanet.URI)) (defn uri [s] (javanet.URI/create s))'
<stdin>:1:18: warning: Unused import URI
<stdin>:1:47: warning: Unresolved namespace javanet.URI. Are you missing a require?
linting took 11ms, errors: 0, warnings: 2
```

This is not because it is a nonexistent class, as other nonexistent classes (e.g. `javaqqq.net.URI`) don't generate the second warning.

**Comment by borkdude** [comment]
@emlyn Hi! I don't think this is a false positive. `import` just means that you can refer to the class unqualified:

```
(ns foo
  (:import java.net.URI))

(defn uri [s]
  (URI/create s))
```

So in your example, the `import` is unused.

**Comment by emlyn** [comment]
Oh yes of course! I got confused because the class I was originally using was not getting put on the classpath.
